app/experimenter/nimbus-ui/src/components/TableOverview/index.tsx

```diff
@@ -3,9 +3,81 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import React from "react";
+import { getExperiment_experimentBySlug } from "../../types/getExperiment";
+import { AnalysisData } from "../../lib/visualization/types";
+import { useConfig } from "../../hooks";
+import { getConfig_nimbusConfig } from "../../types/getConfig";
 
-const TableOverview: React.FunctionComponent = () => {
-  return <div data-testid="table-overview">overview table</div>;
+type TableOverviewProps = {
+  experiment: getExperiment_experimentBySlug;
+  results: AnalysisData["overall"];
+};
+
+type displayConfigOptionsProps =
+  | getConfig_nimbusConfig["firefoxMinVersion"]
+  | getConfig_nimbusConfig["channels"]
+  | getConfig_nimbusConfig["targetingConfigSlug"];
+
+const TableOverview = ({ experiment }: TableOverviewProps) => {
+  const { firefoxMinVersion, channels, targetingConfigSlug } = useConfig();
+
+  return (
+    <div className="mb-5" data-testid="table-overview">
+      <table className="table text-left h5 m-0">
+        <tbody>
+          <tr>
+            <th scope="col">
```

Comment:
I think these should be `<td>` since every cell shouldn't be a table header. Then, "Targeting", "Probe Sets", and "Owner" can be in `<h3 className="h6">` I believe.